pkg/services/services.go

```diff
@@ -431,3 +434,15 @@ func (s *WalletServices) GetBEEF(ctx context.Context, txID string, knownTxIDs []
 
 	return beef, nil
 }
+
+// NLockTimeIsFinal checks if the provided value is a valid nLockTime and whether it is final.
+func (s *WalletServices) NLockTimeIsFinal(ctx context.Context, txOrLockTime any) (bool, error) {
+	result, err := s.nLockTimeIsFinalServices.OneByOne(ctx, txOrLockTime)
+	if err != nil {
+		if errors.Is(err, servicequeue.ErrEmptyResult) {
+			return false, fmt.Errorf("all NLockTimeIsFinal providers failed")
+		}
+		return false, fmt.Errorf("failed to check NLockTimeIsFinal: %w", err)
+	}
+	return result, nil
+}
```

Comment:
I would suggest different (and simpler) approach:
```suggestion
func (s *WalletServices) NLockTimeIsFinal(ctx context.Context, txOrLockTime any) (bool, error) {
	heightProvider := s
	isFinal, err := wdk.NLockTimeIsFinal(ctx, heightProvider, txOrLockTime)
	if err != nil {
		return false, fmt.Errorf("failed to parse nLockTime or final: %w", err)
	}

	return isFinal, nil
}
```

This wouldn't require defining this method for each sub-service. We can just make use of currently defined methods.
What do you think?

If you agree, the changes for bitails/whatsonchain/bhs wouldn't be necessary. 
- And this is good, because if we add some other "service-provider" this method will work out-of-the-box as long as `CurrentHeight` method is provided.